.github/workflows/linux-release.yaml

```diff
@@ -12,23 +16,38 @@ jobs:
       with:
         persist-credentials: false
         fetch-depth: 0
-    - name: Fetch the release asset
-      run: .github/workflows/linux-release.sh
-    - name: Copy linux zip to release for tags
-      uses: canastro/copy-file-action@0.0.2
-      with:
-        source: "dist/rancher-desktop*.zip"
-        target: "dist/rancher-desktop-release.zip"
-    - name: Copy release to s3
+    - name: Populate necessary env vars
+      run: |
+        # get the env vars
+        version_with_v=${GITHUB_REF#"refs/tags/"}
+        release_zip_name="rancher-desktop-linux-${version_with_v}.zip"
+        major_minor=$(echo ${version_with_v} | sed -E 's/v([0-9]+\.[0-9]+)\.[0-9]+.*/\1/g')
+        s3_zip_name="rancher-desktop-linux-${major_minor}.zip"
+
+        # make variables available in subsequent steps
+        echo "version_with_v=$version_with_v" >> $GITHUB_ENV
+        echo "release_zip_name=$release_zip_name" >> $GITHUB_ENV
+        echo "major_minor=$major_minor" >> $GITHUB_ENV
+        echo "s3_zip_name=$s3_zip_name" >> $GITHUB_ENV
+    - name: Fetch the .zip file from release
+      run: |
+        mkdir -p dist
+        curl -L -o "dist/${release_zip_name}" "https://github.com/rancher-sandbox/rancher-desktop/releases/download/${version_with_v}/${release_zip_name}"
```

Comment:
The `release_zip_name` has the version with the v. But, the actual download URL has the version with the `v` for the tag name but without the `v` for the file name... for example, `https://github.com/rancher-sandbox/rancher-desktop/releases/download/v0.7.1/rancher-desktop-0.7.1-linux.zip`

All the file names have the version without the `v`...
<img width="879" alt="Screen Shot 2022-01-20 at 6 03 23 PM" src="https://user-images.githubusercontent.com/62991/150436141-12688422-4f17-46c3-86d6-4ad692552ee4.png">

Reply:
Yes, previous releases have the version without the `v` in the linux .zip file. Here is what I envisioned:

1) `zip_name` is generated using the tag, which is a version string with the`v`
2) the file with the name `zip_name` is uploaded as an artifact
3) when you want to make a release, you simply take the linux .zip file from the workflow run and include it as an asset in the release without changing its name
4) the linux-release.yaml workflow processes the .zip file and its name in order to make things work with OBS

Now that you write this, I realize that having the `v` in the .zip file in a release is inconsistent with the names of the files from other platforms, which isn't great.

I can change this, but it will complicate package.yaml since I will need to process, for example, the `v1.0.0` tag name into `1.0.0`. Alternatively you could just remove the `v` whenever you make a release, but it would be an extra, undocumented, manual step, and I'd still need to change linux-release.yaml.